### PR TITLE
Separate Mobile & Desktop views

### PR DESCRIPTION
**Pull request by josephj**
Refactoring to make CSS easier to understand

### CODE REVIEW

File: client/src/components/QuarterView/Desktop.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          </Row>
+        );
+      })}
+    </Grid>
+  );
+};
+
+const Text = styled.span`
+  flex-grow: 1;
+  overflow: hidden;
```
Comment (james1239090) [inline client/src/components/QuarterView/Desktop.js]: It didn't work on iPad ( 1024*768 )

File: client/src/components/QuarterView/Desktop.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  text-overflow: ellipsis;
```
Comment (james1239090) [inline client/src/components/QuarterView/Desktop.js]: It didn't work on iPad ( 1024*768 )

File: client/src/components/QuarterView/Desktop.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+          .format('YYYY-MM-DD');
+
+        return (
+          <Row key={i} highlighted={day.format('YYYY-MM-DD') === highlightDate}>
+            <Role width={cellWidth}>
+              <AddToCalendar
+                event={icalEvent}
+                listItems={icalitems}
+                buttonTemplate={icalicon}
+                buttonLabel=""
```
[the remark this answers is not in the document]
Comment (josephj) [reply client/src/components/QuarterView/Desktop.js]: It's a really nice feedback. Let's try to put this improvements into the next sprint.

File: client/src/components/QuarterView/Desktop.js (hunk 2 above)
Comment (josephj) [reply client/src/components/QuarterView/Desktop.js]: Will fix

**Pull request merged**